lib/yabeda/sidekiq.rb

```diff
@@ -31,6 +31,8 @@ module Sidekiq
       gauge     :jobs_retry_count,     comment: "The number of failed jobs waiting to be retried"
       gauge     :jobs_dead_count,      comment: "The number of jobs exceeded their retry count."
       gauge     :active_processes,     comment: "The number of active Sidekiq worker processes."
+      gauge     :jobs_latency,         comment: "The job latency: the delay of the job start time in seconds"
```

Comment:
I think that original explanation of what is latency is a bit easier to understand:
https://github.com/mperham/sidekiq/blob/0352ca8fb541dec2caca886aa374e4cbb3c638d9/lib/sidekiq/api.rb#L254-L255

Also you can specify seconds in Yabeda API:
```suggestion
      gauge     :jobs_latency, unit: :seconds, comment: "Time in seconds since the oldest job in the queue was enqueued."
```